misc/ballerina-bindgen/src/main/java/org/ballerinalang/bindgen/utils/MvnResolverUtils.java

```diff
@@ -173,4 +173,8 @@ private static String getModuleName(Path projectRoot, String outputPath) {
         }
         return null;
     }
+
+    public static void setOutStream(PrintStream outStream) {
+        MvnResolverUtils.outStream = outStream;
```

Comment:
@IrushiL Its good to use setters for a non static property, since this is a static property can't we just pass outStream to relevant static methods as a param?

Reply:
Refactored the code and made it a private variable instantiated through the constructor.